CatShelter.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "PetCat.h"
+#include "Human.h"
+
+struct CatShelter
+{
+    PetCat msDoris;
+    PetCat::CatCollar catCollar;
+    int roomForCat {1000};
+
+    CatShelter();
+    ~CatShelter();
+
+    int AdmitNewCat(PetCat& newCat, PetCat::CatCollar newCollar);
+
+    int getCatAdopted(const PetCat& newCat, const Human catAdopter);
```

Comment:
Don't pass by copy, pass by const reference.
`Human` can be forward declared and the header file can be included in the cpp file
- [ ] revise